pkgs/top-level/python-packages.nix

```diff
@@ -1320,6 +1320,8 @@ in {
 
   brother = callPackage ../development/python-modules/brother { };
 
+  brother_ql = callPackage ../development/python-modules/brother_ql { };
```

Comment:
```suggestion
  brother-ql = callPackage ../development/python-modules/brother-ql { };
```

Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)

Reply:
Fixed link: https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

Reply:
Done, thanks!